### DISCUSSION TITLE
jsonwebtoken 8.5.1 dependecy -  High-severity Vulnerability

### DISCUSSION THREAD

**Discussion by castafab** [vercel/next.js#44770] [Help] [answered]
Hi, there are plans for NextJs to update the jsonwebtoken dependency from 8.5.1 to the 9.0.0 that contains the fix to the vulnerabiliy https://github.com/advisories/GHSA-27h2-hvpr-p74q?

**Comment by icyJoseph** [comment, marked as answer]
https://github.com/vercel/next.js/pull/44748 it has been patched ~ expect it in the next minor release

**Reply by bran-ch** [reply]
@icyJoseph Could this be applied to NextJS V12 as well?
